src/generic/scrlwing.cpp

```diff
@@ -1554,10 +1554,10 @@ wxSize wxScrolledT_Helper::FilterBestSize(const wxWindow *win,
         wxSize minSize = win->GetMinSize();
 
         if ( ppuX > 0 )
-            best.x = minSize.x + wxSystemSettings::GetMetric(wxSYS_VSCROLL_X);
+            best.x = minSize.x + wxSystemSettings::GetMetric(wxSYS_VSCROLL_X, const_cast<wxWindow*>(win));
```

Comment:
I think it might be better to take `const wxWindow*` parameter as querying a metric doesn't really modify the window, conceptually. And this would avoid the casts here.

Reply:
Done.